### PR TITLE
external sso

### PR DESCRIPTION
**Pull request by ultraschuppi**
## What is the purpose of the change

adding a brief intro on external sso support for `next`

closes https://github.com/camunda/camunda-platform-docs/issues/1133

## Are there related marketing activities

nothing i am aware of 

## When should this change go live?

next drumbeat release

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by felix-mueller** [comment]
lgtm

**Review by felix-mueller — DISMISSED** [review]

File: docs/components/console/manage-organization/external-sso.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+id: external-sso
+title: Connect your IDP with Camunda Platform
+---
+
+## Connecting your Identity Provider with Camunda Platform
```
Comment (meyerdan) [inline docs/components/console/manage-organization/external-sso.md:6]: @ultraschuppi should we include the Keyword "SSO" in the title?

Comment (ultraschuppi) [reply docs/components/console/manage-organization/external-sso.md:7]: good point! done in https://github.com/camunda/camunda-platform-docs/pull/1338/commits/f6c482e6a5b40a4cdf47008feb9a1200fcaeb3b6

**Review by akeller — APPROVED** [review]
I made some small fixes for capitalization, typo, etc.

**Pull request merged**